gettsim/pre_processing/policy_for_date.py

```diff
@@ -49,61 +50,146 @@
 
 
 def get_policies_for_date(policy_date, groups="all"):
-    if isinstance(policy_date, str):
-        policy_date = pd.to_datetime(policy_date)
+    """Get the state of the policy system for a particular date.
+
+    The function returns time dependent policy reforms (functions) as well as the
+    policy parameters for the current policy system.
+
+    Parameters
+    ----------
+    policy_date : datetime.date, str, int
+                  The date for which the policy system is set up.
+    groups : list, str
+             The group or a list of groups which parameters are loaded. Default is
```

Comment:
What are example of group names? How can the user retrieve all names?

Reply:
Solved!